components/Card/card.css

```diff
@@ -0,0 +1,110 @@
+/* All Cards */
+
+.card, .card-badge{
+	border-radius: 0.5rem;
+	font-family: var(--font-secondary);
+}
+.card{
+	position:relative;
```

Comment:
You can use "beautify file extension" to beautify your file. So, that consistent spacing and indentation can be achieved.